config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "publicProps":
```

Comment:
The json is more complex than the one in the other project. As this is a config file for users, it seems that we want to maximize the ease of use and understanding. I also expect that all the properties we want to add here will be in publicProps, as they will be generic metadata for the project. Is it necessary therefore to have this main json category?

Reply:
It may look simpler again once we use the app config instead of the runtime config. If not, there are ways of importing the json so that we hide these extra layers